sys/net/routing/rpl/rpl.c

```diff
@@ -58,6 +56,12 @@ mutex_t rpl_send_mutex = MUTEX_INIT;
 msg_t rpl_msg_queue[RPL_PKT_RECV_BUF_SIZE];
 char rpl_process_buf[RPL_PROCESS_STACKSIZE];
 uint8_t rpl_buffer[BUFFER_SIZE - LL_HDR_LEN];
+ipv6_addr_t mcast;
+timex_t rt_time;
+vtimer_t rt_timer;
+
+static void dao_handle_send(rpl_dodag_t *dodag);
+static void rpl_update_routing_table(void);
```

Comment:
For static functions I would vote for omitting the prefix or make it `_rpl_update_routing_table()`